platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/Capture.kt

```diff
@@ -206,6 +209,12 @@ object Capture {
             logger()?.startNewSession()
         }
 
+        @JvmStatic
+        fun shutdown() {
+            logger()?.shutdown()
```

Comment:
I wonder if these need to be flipped, I think we'd crash if something tried to do something with the loggerid while shutdown is being called, so seems safer to pretend like we are shut down earlier?